keras/utils/training_utils.py

```diff
@@ -88,6 +88,10 @@ def multi_gpu_model(model, gpus):
     import tensorflow as tf
 
     target_devices = ['/cpu:0'] + ['/gpu:%d' % i for i in range(gpus)]
+    target_devices = list(map(
+        lambda x: tf.DeviceSpec.from_string(x).to_string() if x.find('gpu') != -1 else x,
```

Comment:
- Why exclude the CPU from the normalization process?
- Why are the available devices not similarly normalized?